test/utils/runners.go

```diff
@@ -1194,6 +1194,12 @@ func CreatePodWithPersistentVolume(ctx context.Context, client clientset.Interfa
 		// PVs are cluster-wide resources.
 		// Prepend a namespace to make the name globally unique.
 		pv.Name = fmt.Sprintf("%s-%s", namespace, pv.Name)
+		pvs := pv.Spec.PersistentVolumeSource
+		if pvs.CSI != nil {
+			pvs.CSI.VolumeHandle = pv.Name
+		} else if pvs.AWSElasticBlockStore != nil {
```

Comment:
the non-csi volume plugins are slowly getting removed, so I prefer we keep them out of the benchmarks

Reply:
Done, changed to use only the CSI